Form/DataTransformer/LegacyModelsToArrayTransformer.php

```diff
@@ -17,11 +17,9 @@
 use Symfony\Component\Form\Exception\UnexpectedTypeException;
 
 /**
- * Class ModelsToArrayTransformer.
+ * This class should be used with Symfony <2.7 only and will be deprecated with 3.0.
```

Comment:
Can I overhand this to you, you are more familar with this 😄  and anyway, this is put of scope of this PR